relay.go

```diff
@@ -524,13 +530,13 @@ func (r *Relayer) handleCallReq(f *lazyCallReq) (shouldRelease bool, _ error) {
 	sent, failure := relayToDest.destination.Receive(f.Frame, requestFrame)
 	if !sent {
 		r.failRelayItem(r.outbound, origID, failure, errFrameNotSent)
-		return _relayNoRelease, nil
+		return _relayShouldRelease, nil
 	}
-	return _relayNoRelease, nil
+	return !sent, nil
```

Comment:
since we handle `!sent` above, `!sent` will always be false here, so this should just be `return false, nil` or using the constants: `return _relayNoRelease, nil`. simpler to follow